src/client/components/pages/entities/identifiers.js

```diff
@@ -22,13 +22,16 @@ import React from 'react';
 
 
 function EntityIdentifiers({identifiers, identifierTypes}) {
+	const editLink = typeof window !== 'undefined' ? `${window.location.pathname}/edit` : '';
+
 	return (
 		<div>
 			<h2>Identifiers</h2>
 			{
-				identifiers &&
-				identifierTypes.sort((a, b) => a.label.localeCompare(b.label)).map((type) => {
-					const identifierValues =
+
+				identifiers && identifiers.length > 0 ?
```

Comment:
We can shorten this just a bit 
```suggestion
				identifiers?.length > 0 ?
```